examples/extensions/commands/clean/cmd_clean.py

```diff
@@ -1,11 +1,31 @@
 from conan.api.conan_api import ConanAPI
+from conan.internal.conan_app import ConanBasicApp
```

Comment:
But adding internal imports will also be bad, and this makes it inmediately available to users.
An ``internal`` import should never be in any example or docs, I think the way to go is change it directly to use the ``list`` api instead